types/array_basic.go

```diff
@@ -1,27 +1,30 @@
 package types
 
 import (
+	"github.com/dgraph-io/ristretto"
+	"github.com/minio/highwayhash"
 	"reflect"
 	"sync"
-	"time"
-
-	"github.com/karlseguin/ccache"
-	"github.com/minio/highwayhash"
 )
 
 // BasicArraySizeCache for HashTreeRoot.
-const BasicArraySizeCache = 100000
+const BasicArraySizeCache = 1e6
```

Comment:
why 1e6?

Reply:
good point, 1 million may be overkill for a cache of this purpose